api/v1alpha1/edgeplaybook_types.go

```diff
@@ -0,0 +1,116 @@
+/*
+Copyright 2021.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// EdgePlaybookSpec defines the desired state of EdgePlaybook
+type EdgePlaybookSpec struct {
+	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+	// Important: Run "make" to regenerate code after modifying this file
+
+	DeviceSelector *metav1.LabelSelector `json:"deviceSelector,omitempty"`
+
+	// The ansible playbook command to execute
+	AnsiblePlaybookCmd *AnsiblePlaybookCmd `json:"ansiblePlaybookCmd,omitempty"`
+
+	LogCollection map[string]*LogCollectionConfig `json:"logCollection,omitempty"`
+}
+
+type AnsiblePlaybookCmd struct {
+	// username who execute the playbook
+	User string `json:"user,omitempty"`
+	// Ansible binary file
+	BinaryPath string `json:"binaryPath,omitempty"`
```

Comment:
This can be useful both to specify the version to use and to specify a non standard ansible binary path.
We don't have customer requirements, so I'll try to figure out what the scenario may look like.

We can simply the CRD, at least in this step, by removing this field.
WDYT?

Reply:
IMO at this stage we should simplify as much as possible - to the bare minimum.

Reply:
+1 for removing it.